### PR TITLE
Login styling

### PR DESCRIPTION
**Pull request by SavageWilliam**
issue #60 

image

### CODE REVIEW

**Comment by bradreeder** [comment]
'welcome' not 'wellcome'. classic @nogainbar , I sense your hand in this. 😄

**Comment by des-des** [comment]
@SavageWilliam you need add issue number to your commits.

Either
use git reset and recommit
or do an interactive rebase
With `git reset`

image


Once you have recommit with issue numbers attached to commit messages you will need to force push

**Comment by des-des** [comment]
@SavageWilliam If you think I am being too mean, re-assign and ill let it go, thats ok.

**Comment by SavageWilliam** [comment]
@des-des No no it's good. I'm happy to be improving my questionable git flow! I needed to make another change anyway to have recommited with the issue

**Comment by des-des** [comment]
ok the idea is add add issue number to *every* commit message. The steps I outlined would allow you to add issue numbers to your old commits..

**Comment by des-des** [comment]
@SavageWilliam reassign when you have understood my comment

**Comment by SavageWilliam** [comment]
@des-des I attempted to use git reset and nothing happened...

I wasn't aware that the issue number must be add to EVERY commit? I'll do that from now. Would it be alright to merge this PR for now? Just as we have a few hours left to complete the prototype for this sprint and a lot to do still :smile:

**Comment by des-des** [comment]
@SavageWilliam as long as you understand I am happy!

File: login.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const letsGo = document.getElementById('letsGo');
```
Comment (des-des) [inline login.js]: `letsGo` has global scope.
 1. wrap file in function or
 2. just go `document.getElementById('letsGo').addEventListener('click', () => {`

**Comment by des-des** [comment]
@SavageWilliam have reviewed now, just one small change! :smile:

**Comment by SavageWilliam** [comment]
@des-des  :D thanks for you're code reviews! Hopefully this is the one!

**Pull request merged**